### PR TITLE
19.02 Overview of MVP architecture in Android(Audio file)

### LINKED ISSUE #7357: Android-Kotlin : 19.02 Overview of MVP architecture in Android.
<hr>

## Description :scroll:

*  Give a step-by-step tutorial in Kotlin along with a brief and clear explanation.
*  Required Documentation or Video or Audio.

<hr>

## Domain of Contribution :bar_chart:

- [x] Android Dev (kotlin)

<hr>

<hr>

<hr>

## Please Read it Carefully Before Contributing
* The file structure should look like `Android_Development_With_Kotlin  -> 19 Architecture -> <file name>`.
* You have to push your PR to the Android_Development_With_Kotlin branch only.
* You can take help to the references but don't simply try to copy it. Just explain in your own word.
* Your code should be written only in Kotlin.
* This issue is for GWOC'21 contributors only.
* You need to push in a single commit. If there exists more than one commit then you need to squash your commits into one commit.
* Will assign you on a first come first serve basis.
* Please try to complete your assigned task within 7 days. If you need some more time then, please inform mentors and supervisors.
* If you have two previously assigned issues on Android-Kotlin pending then please first complete them.
> Supervisor @rahulsain 

## Happy Contributing. Best of Luck.:sparkling_heart:

### PR DESCRIPTION
**Pull request by Parth-001-git**
<hr>

## Description 📜

In this repo we will get to know about  Overview of MVP architecture in Android
Fixes #7357

<hr>

## Type of change 📝

- [x] Audio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [ ] Doc#umentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [x] Android Dev (Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by Parth-001-git** [comment]
@rahulsain  because there was no new file related to the 19 number I knowingly have to create another one please review.

**Comment by rahulsain** [comment]
The naming of the main folder should be referred from the roadmap.
1 PR should solve only 1 issue, so make the changes that solves the mentioned issue one at a time

File: Android_Development_with_Kotlin/05.1 Basic Implementation of Views in Android.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## 05.1 Basic Implementation of Views in Android
+[Basic Implementation of Views in Android]( https://drive.google.com/drive/u/0/folders/1Y2rTWJnhnOuRWRMPmEo5gVh4tjsBZDM2)
```
Comment (rahulsain) [inline Android_Development_with_Kotlin/05.1 Basic Implementation of Views in Android.md:2]: this doesnt fixed the mentioned issue, hence this file is not required in this PR

File: Chronometer in Android.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
+_Android ChronoMeter is user interface control which shows timer in the view. We can easily start up or down counter with base time using the chronometer widget. By default, start() method can assume base time and starts the counter.
+Generally, we can create use ChronoMeter widget in XML layout but we can do it programmatically also._
```
Comment (rahulsain) [inline Chronometer in Android.md:2]: this doesnt fixed the mentioned issue, hence this file is not required in this PR

File: Android_Development_with_Kotlin/19. Overview of MVP architecture in Android/19.02 Overview of MVP architecture in Android.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#  Overview of MVP architecture in Android (Audio file)
+[MVP architecture in Android ](https://drive.google.com/file/d/1yons4_a1zp-TUSe2AqQglbtjk-Ak5-XJ/view?usp=sharing)
```
Comment (rahulsain) [inline Android_Development_with_Kotlin/19. Overview of MVP architecture in Android/19.02 Overview of MVP architecture in Android.md:2]: change folder structure from `Android_Development_with_Kotlin/19. Overview of MVP architecture in Android/19.02 Overview of MVP architecture in Android.md` to `Android_Development_with_Kotlin/19. Architecture/19.02 Overview of MVP architecture in Android.md`

**Review by rahulsain — CHANGES_REQUESTED** [review]

**Pull request closed without merging**